changelog/unreleased/pull-2750

```diff
@@ -0,0 +1,5 @@
+Enhancement: Add --min-packsize and --file-read-concurrency
```

Comment:
maybe mention --save-blob-concurrency :wink:

Reply:
I plan to.  I haven't touched docs yet either, so I was going to handle all of that in the next couple commits, when I get some time.